<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>View</class>
 <widget class="QMainWindow" name="View">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1232</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>View</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="GraphicsView">
    <property name="geometry">
     <rect>
      <x>-50</x>
      <y>-180</y>
      <width>1231</width>
      <height>771</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <property name="indent">
     <number>0</number>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1232</width>
     <height>22</height>
    </rect>
   </property>
   <property name="nativeMenuBar">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionClear_all"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionCalibrate"/>
    <addaction name="actionHelp"/>
   </widget>
   <widget class="QMenu" name="menuPaint_Settings">
    <property name="title">
     <string>Paint Settings</string>
    </property>
    <addaction name="actionColorpicker"/>
    <addaction name="actionStrokewidth"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuPaint_Settings"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionPaint"/>
   <addaction name="actionErase"/>
   <addaction name="actionRed"/>
   <addaction name="actionBlue"/>
   <addaction name="actionYellow"/>
   <addaction name="actionThick"/>
   <addaction name="actionMedium"/>
   <addaction name="actionThin"/>
   <addaction name="actionUndo"/>
   <addaction name="actionRedo"/>
  </widget>
  <action name="actionNeu">
   <property name="text">
    <string>Clear all</string>
   </property>
   <property name="toolTip">
    <string>Erstelle ein neues Bild</string>
   </property>
  </action>
  <action name="actionOeffnen">
   <property name="text">
    <string>Öffnen</string>
   </property>
   <property name="toolTip">
    <string>Öffne ein bereits vorhandenes Bild</string>
   </property>
  </action>
  <action name="actionSpeichern">
   <property name="text">
    <string>Speichern</string>
   </property>
   <property name="toolTip">
    <string>Speichere ein Bild ab </string>
   </property>
  </action>
  <action name="actionExportieren">
   <property name="text">
    <string>Exportieren</string>
   </property>
   <property name="toolTip">
    <string>Exportiere ein Bild</string>
   </property>
  </action>
  <action name="actionKalibrieren">
   <property name="text">
    <string>Kalibrieren</string>
   </property>
   <property name="toolTip">
    <string>Kalibriere die Anwendung</string>
   </property>
  </action>
  <action name="actionHilfe">
   <property name="text">
    <string>Hilfe</string>
   </property>
   <property name="toolTip">
    <string>Hilfe zur Anwendung</string>
   </property>
  </action>
  <action name="actionrot">
   <property name="icon">
    <iconset>
     <normaloff>rot.PNG</normaloff>rot.PNG</iconset>
   </property>
   <property name="text">
    <string>rot</string>
   </property>
  </action>
  <action name="actiongruen">
   <property name="icon">
    <iconset>
     <normaloff>gruen.PNG</normaloff>gruen.PNG</iconset>
   </property>
   <property name="text">
    <string>grün</string>
   </property>
   <property name="toolTip">
    <string>grün</string>
   </property>
  </action>
  <action name="actionblau">
   <property name="icon">
    <iconset>
     <normaloff>blau.PNG</normaloff>blau.PNG</iconset>
   </property>
   <property name="text">
    <string>blau</string>
   </property>
  </action>
  <action name="actiongelb">
   <property name="icon">
    <iconset>
     <normaloff>gelb.PNG</normaloff>gelb.PNG</iconset>
   </property>
   <property name="text">
    <string>gelb</string>
   </property>
  </action>
  <action name="actionweiss">
   <property name="icon">
    <iconset>
     <normaloff>weiss.PNG</normaloff>weiss.PNG</iconset>
   </property>
   <property name="text">
    <string>weiß</string>
   </property>
  </action>
  <action name="actionDuenn">
   <property name="icon">
    <iconset>
     <normaloff>icons/duenn.PNG</normaloff>icons/duenn.PNG</iconset>
   </property>
   <property name="text">
    <string>Dünn</string>
   </property>
  </action>
  <action name="actionMittel">
   <property name="icon">
    <iconset>
     <normaloff>icons/mittel.PNG</normaloff>icons/mittel.PNG</iconset>
   </property>
   <property name="text">
    <string>Mittel</string>
   </property>
  </action>
  <action name="actionDick">
   <property name="icon">
    <iconset>
     <normaloff>icons/dick.PNG</normaloff>icons/dick.PNG</iconset>
   </property>
   <property name="text">
    <string>Dick</string>
   </property>
  </action>
  <action name="actionschwarz">
   <property name="icon">
    <iconset>
     <normaloff>schwarz.PNG</normaloff>schwarz.PNG</iconset>
   </property>
   <property name="text">
    <string>schwarz</string>
   </property>
  </action>
  <action name="actionPaint">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>icons_win/zeichnen.PNG</normaloff>icons_win/zeichnen.PNG</iconset>
   </property>
   <property name="text">
    <string>Paint</string>
   </property>
  </action>
  <action name="actionErase">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>icons_win/radierer.PNG</normaloff>icons_win/radierer.PNG</iconset>
   </property>
   <property name="text">
    <string>Erase</string>
   </property>
  </action>
  <action name="actionThick">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>icons_win/dick.PNG</normaloff>icons_win/dick.PNG</iconset>
   </property>
   <property name="text">
    <string>Thick</string>
   </property>
  </action>
  <action name="actionMedium">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>icons_win/mittel.PNG</normaloff>icons_win/mittel.PNG</iconset>
   </property>
   <property name="text">
    <string>Medium</string>
   </property>
  </action>
  <action name="actionThin">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>icons_win/duenn.PNG</normaloff>icons_win/duenn.PNG</iconset>
   </property>
   <property name="text">
    <string>Thin</string>
   </property>
  </action>
  <action name="actionRed">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>icons_win/rot.PNG</normaloff>icons_win/rot.PNG</iconset>
   </property>
   <property name="text">
    <string>Red</string>
   </property>
  </action>
  <action name="actionBlue">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>icons_win/blau.PNG</normaloff>icons_win/blau.PNG</iconset>
   </property>
   <property name="text">
    <string>Blue</string>
   </property>
  </action>
  <action name="actionYellow">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>icons_win/gelb.PNG</normaloff>icons_win/gelb.PNG</iconset>
   </property>
   <property name="text">
    <string>Yellow</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset>
     <normaloff>icons_win/undo.PNG</normaloff>icons_win/undo.PNG</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset>
     <normaloff>icons_win/redo.PNG</normaloff>icons_win/redo.PNG</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
  <action name="actionClear_all">
   <property name="text">
    <string>Clear all</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionCalibrate">
   <property name="text">
    <string>Calibrate</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionColorpicker">
   <property name="icon">
    <iconset>
     <normaloff>icons_win/Farbe.PNG</normaloff>icons_win/Farbe.PNG</iconset>
   </property>
   <property name="text">
    <string>Colorpicker</string>
   </property>
  </action>
  <action name="actionStrokewidth">
   <property name="icon">
    <iconset>
     <normaloff>icons_win/Strichdicke.PNG</normaloff>icons_win/Strichdicke.PNG</iconset>
   </property>
   <property name="text">
    <string>Strokewidth</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
